build/lib/release.sh

```diff
@@ -296,8 +303,27 @@ function kube::release::build_hyperkube_image() {
   fi
 }
 
-# This will take binaries that run on master and creates Docker images
-# that wrap the binary in them. (One docker image per binary)
+function kube::release::build_conformance_image() {
+  local -r arch="$1"
+  local -r registry="$2"
+  local -r version="$3"
+  local -r save_dir="${4-}"
+  kube::log::status "Building conformance image for arch: ${arch}"
+  ARCH="${arch}" REGISTRY="${registry}" VERSION="${version}" \
+    make -C cluster/images/conformance/ build >/dev/null
+
+  local conformance_tag="${registry}/conformance-${arch}:${version}"
+  if [[ -n "${save_dir}" ]]; then
+    "${DOCKER[@]}" save "${conformance_tag}" > "${save_dir}/conformance-${arch}.tar"
+  fi
+  if [[ -z "${KUBE_DOCKER_IMAGE_TAG-}" || -z "${KUBE_DOCKER_REGISTRY-}" ]]; then
```

Comment:
I'm pretty sure we can always call `rmi` here - we use the tarballs to push to gcr.io these days, rather than pushing images directly. (The conditionals are basically legacy stuff I forgot to clean up after https://github.com/kubernetes/kubernetes/pull/47939.)

Reply:
- ok removing the test binaries 
- cool, removing the if condition around `rmi`